htdocs/core/class/hookmanager.class.php

```diff
@@ -155,6 +160,26 @@ public function initHooks($arraycontext)
 	 */
 	public function executeHooks($method, $parameters = array(), &$object = '', &$action = '')
 	{
+        global $debugbar;
+		if (is_object($debugbar) && get_class($debugbar) === 'DolibarrDebugBar') {
```

Comment:
Do you use this test ti detect "if debugbar module is enabled" ?
If yes can you try with just
If (isModEnabled('debugbar')) {